bin/console_runner.js

```diff
@@ -176,10 +176,11 @@ function start(options)
 
             fs.mkdir(outDir, 0o775, function(){
 				var outPath = libpath.join(outDir, testRunner.uuid+'.log');
-				fs.writeFile(outPath, output);
```

Comment:
Yeah, this makes some sense. This was a sort of "It does not matter when this log file finishes writing". So we didn't pass any callback at all. I guess now that the whole thing is rewritten to use promises, it uses value of the final argument to figure out if it should return a promise or call a callback. Thus the deprecation.